src/components/qas/styles.module.scss

```diff
@@ -28,8 +28,8 @@
   .qa-review-label-post,
   .calculator-label,
   p {
-    font-size: 16px !important; /* stylelint-disable-line declaration-no-important *//* stylelint-disable-line declaration-no-important *//* stylelint-disable-line declaration-no-important */
-    font-weight: $medium !important; /* stylelint-disable-line declaration-no-important *//* stylelint-disable-line declaration-no-important */
+    font-size: 16px !important; /* stylelint-disable-line declaration-no-important */ /* stylelint-disable-line declaration-no-important */ /* stylelint-disable-line declaration-no-important */
+    font-weight: $medium !important; /* stylelint-disable-line declaration-no-important */ /* stylelint-disable-line declaration-no-important */
```

Comment:
Not sure what's going on here but got too many disable linter rule comments goin on here.  Maybe it doesn't like neighboring lines having the disable-line style comment?  Try the other style of comment where you tell it when to start and stop ignoring the rule?  [More info here](https://stylelint.io/user-guide/ignore-code).